### ISSUE TITLE
Deployment issues

### ISSUE THREAD

**Issue by kofler86** [InsiderPhD/Generic-University#4]
Hi, 

Could you be update the instalation guide? are there any prerequisites for that? I tried to clone it and follow instructions of my kali linux but I failed with database bindind :) 

I assume you need some laravel enviroment configured first.
Please let me know. 

BTW I love your YT channel :)

**Comment by InsiderPhD** [comment]
Very fair comment, I will prioritise dockerising (https://github.com/InsiderPhD/Generic-University/issues/1).
For help right now I believe you will need PHP, composer + a database, I'll make sure to make this clearer on the installation!

**Comment by kofler86** [comment]
Yes I need php and php-xml installed, suprisingly it didn't come with php. If you would like to chat about that let me know, you can find me on Linkedin as Maciej Kofel

**Comment by InsiderPhD** [comment]
I've attempted this with f0c922cf411e19d0a1a2ce8d925f77a8c316b6fa - but it will likely need docker for a easy install, making this top priority
